docs/modules/ROOT/pages/advanced-how-tos/keep-remote-pipelines-up-to-date.adoc

```diff
@@ -0,0 +1,61 @@
+= Maintaining remote Tekton pipelines up to date
+
+== The problem
+
+Although Tekton remote pipelines is a huge improvement for having a centralized and shared repo for Pipelines and Tasks, it comes with a drawback; the Pipeline CRDs will not be stored in the Konflux Component repository anymore, so MintMaker will not be able to update the TaskRef digests for us.
+
+== The solution
+To workaround the above issue, the suggested approach is to onboard the repository where the shared pipelines are stored as a Konflux Component, so that Konflux and MintMaker will be able to discover and update digests for us.
+
+.*Prerequisites*
+
+* Having a repository where to store shared Pipelines and Tasks.
+
+[IMPORTANT]
+====
+If the repository is private, be sure to have a Secret in place on Konflux to be able to clone it
+====
+
+.*Procedures*
+
+. Add Tekton Pipelines and Tasks in the repository. A suggested repository structure would be:
+
++
+[source,shell]
+----
+.
+├── .tekton
+│   ├── <repository-name>-pipelines-pull-request.yaml # Used by Konflux to listen to pull requests
+│   └── <repository-name>-pipelines-push.yaml # Used by Konflux to listen to pushes
+├── pipelines
+│   ├── remote-pipeline1.yaml
+│   └── remote-pipeline2.yaml
+├── tasks
+│   ├── remote-task1.yaml
+│   └── remote-task2.yaml
```

Comment:
What is the difference between all of these files? What should go into each of these?

Reply:
The several remote pipelines that could be run from different components. I.E.
- multi aarch pipeline
- hermetic pipeline
- pipeline with disabled checks or a very few tasks in it
- pipeline with pre-fetch enabled
- custom pipeline with untrusted Tasks
and so on.